module/VuFind/src/VuFind/Db/Service/LoginTokenService.php

```diff
@@ -121,7 +149,12 @@ public function matchToken(array $token): ?LoginTokenEntityInterface
      */
     public function deleteBySeries(string $series, ?int $currentTokenId = null): void
     {
-        $this->getDbTable('LoginToken')->deleteBySeries($series, $currentTokenId);
+        $dql = 'DELETE FROM ' . $this->getEntityClass(LoginToken::class) . ' lt '
+            . 'WHERE lt.series = :series'
+            . 'AND lt.id != :currentTokenId';
```

Comment:
Have you tested that this works correctly when `$currentTokenId` is set to null? The old code only added this clause conditionally when `$currentTokenId` was non-null. Maybe it's okay to do it this way, but we should verify if we haven't already.